src/components/network/config-tables.js

```diff
@@ -214,6 +214,27 @@ export const TABLES_DEFINITIONS = {
                 fractionDigits: 1,
                 canBeInvalidated: true,
             },
+            {
+                id: 'LoadTapChangingCapabilities',
+                dataKey: 'loadTapChangingCapabilities',
+                boolean: true, // new param to indicate that this cell data is boolean(true/false)
```

Comment:
It's a new param for now then it will not :smile: 
I think you should remove this comment, we understand that this column if a boolean type.